packages/platform/types/event-types/event-types_2024_06_14/outputs/event-type.output.ts

```diff
@@ -419,25 +420,37 @@ export class TeamEventTypeOutput_2024_06_14 extends BaseEventTypeOutput_2024_06_
   @DocsProperty()
   hideCalendarEventDetails?: boolean;
 
+  @IsObject()
+  @IsOptional()
+  @DocsProperty()
+  team!: Team | null;
+}
+
+class Team {
+  @IsInt()
+  @IsOptional()
+  @DocsProperty()
+  id?: number | null;
```

Comment:
Team id will be defined if the team is defined, so check prisma schema for the team and see which properties are defined and not.